content/en/blog/_posts/2022-08-22-Announcing-Auto-Refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: blog
+title: "Announcing the Auto-refreshing Official Kubernetes CVE Feed"
```

Comment:
```suggestion
title: "Announcing the Official Kubernetes CVE Feed"
```
I think readers will take for granted that we update it automatically.

Reply:
I could be wrong, but wanted to preemptively explicitly mention, that the feed will be refreshed periodically by putting in the title as I can imagine that being the most frequently asked question.

Reply:
I see the challenge. Is “live feed” too much of a promise? “automatically refreshing” is a lot of syllables.

We could explain in the text that the feed is updated multiple times each day, and repeat that explanation lower in the article text, and then hope that folks read the article.